node/test/go/dev-mode/test/lib/create-core-resources.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const path = require('path');
+const fsp = require('fs').promises;
+const { Lambda } = require('@aws-sdk/client-lambda');
+const { IAM } = require('@aws-sdk/client-iam');
+const { STS } = require('@aws-sdk/client-sts');
+const log = require('log').get('test');
+const awsRequest = require('../utils/aws-request');
+const basename = require('./basename');
+
+const createLayer = async ({ layerName, filename }) => {
+  log.info('Publishing layer (%s) to AWS', filename);
+  const arn = (
+    await awsRequest(Lambda, 'publishLayerVersion', {
+      LayerName: layerName,
+      Content: { ZipFile: await fsp.readFile(filename) },
+    })
+  ).LayerVersionArn;
+  log.info('Layer ready %s', arn);
+  return arn;
+};
+
+const createLayers = async (config, layerTypes) => {
+  return Promise.all(
+    Array.from(layerTypes, async (layerType) => {
+      switch (layerType) {
+        case 'nodeExternal':
+          if (process.env.TEST_EXTERNAL_LAYER_FILENAME) {
+            config.layerExternalArn = await createLayer({
+              layerName: `${basename}-external`,
+              filename: path.resolve(process.env.TEST_EXTERNAL_LAYER_FILENAME),
+              skipBuild: true,
+            });
+            return;
+          }
+          config.layerExternalArn = await createLayer({
```

Comment:
So at this point this test just loads external extension without internal, right?

I think it'll be good we also load internal, as it'll reflect real-world scenario, and we'll be sure we're covering the right way all eventual side effects that internal extension introduces

Reply:
Yeah we might actually need one set of tests where we only have the dev mode extension and then another set of tests that have both internal and external extension loaded 🤔 

Would we want to add this test now even though we don't have the external and internal integration set up yet?

Reply:
If the intention is to include dev-mode extension independently out of our SDK, then let's skip it.

I somewhat assumed that our approach is to have them altogether from start, but that's not the case I take

Reply:
Yeah exactly 👍 The goal right now is to be able to use this external extension independently from the log subscription and sdk extension.

The next step for this extension is to integrate with the sdk in some fashion like I DMed you about last week. But even that integration will be optional so when we get to implementing that I will need to add tests with and without the SDK.